src/components/Patient/PatientHome.tsx

```diff
@@ -657,15 +658,16 @@ export const PatientHome = (props: {
                 {patientData.last_consultation?.new_discharge_reason ===
                   DISCHARGE_REASONS.find((i) => i.text == "Expired")?.id && (
                   <div>
-                    <ButtonV2
+                    <Button
                       id="death-report"
-                      className="my-2 w-full"
+                      className="my-2 w-full font-semibold"
```

Comment:
let's not alter the weight inside the button. let's stick with what the variant provides.

```suggestion
                      className="my-2 w-full"
```